concrete/single_pages/dashboard/system/seo/codes.php

```diff
@@ -19,17 +19,42 @@
 
     <div class="form-group">
         <?= $form->label('tracking_code_header', t('Header Tracking Codes')) ?>
-        <?= $form->textarea('tracking_code_header', $tracking_code_header, ['style' => 'height: 250px;', 'class' => 'form-control font-monospace', 'spellcheck' => 'false']) ?>
+        <?= $form->textarea('tracking_code_header_input', $tracking_code_header, ['style' => 'height: 250px;', 'class' => 'form-control font-monospace', 'spellcheck' => 'false']) ?>
+        <?= $form->hidden('tracking_code_header', '') ?>
     </div>
 
     <div class="form-group">
         <?= $form->label('tracking_code_footer', t('Footer Tracking Codes')) ?>
-        <?= $form->textarea('tracking_code_footer', $tracking_code_footer, ['style' => 'height: 250px;', 'class' => 'form-control font-monospace', 'spellcheck' => 'false']) ?>
+        <?= $form->textarea('tracking_code_footer_input', $tracking_code_footer, ['style' => 'height: 250px;', 'class' => 'form-control font-monospace', 'spellcheck' => 'false']) ?>
+        <?= $form->hidden('tracking_code_footer', '') ?>
     </div>
 
     <div class="ccm-dashboard-form-actions-wrapper">
         <div class="ccm-dashboard-form-actions">
-            <button type="submit" class="btn btn-primary float-end"><?= t('Save') ?></button>
+            <button type="submit" id="tracking-code-form-button" class="btn btn-primary float-end"><?= t('Save') ?></button>
         </div>
     </div>
 </form>
+
+<script>
+    document.addEventListener('DOMContentLoaded', function(event) {
+        var trackingCodeForm = document.getElementById("tracking-code-form-button");
+        var trackingCodeHeaderInput = document.getElementById("tracking_code_header_input");
+        var trackingCodeFooterInput = document.getElementById("tracking_code_footer_input");
+        var trackingCodeHeader = document.getElementById("tracking_code_header");
+        var trackingCodeFooter = document.getElementById("tracking_code_footer");
+
+        trackingCodeForm.addEventListener("click", function(e) {
```

Comment:
I'd hook the `submit` event of the `tracking-code-form` form instead...

Reply:
I think I put on the button click because I had to do it on the block for some reason, but it works fine on the form submit event - I've updated the PR.